pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -166,9 +166,17 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 		return f, nil
 	}
 
-	pluginConfig := make(map[string]*runtime.Unknown, 0)
+	pluginConfig := make(map[string]*runtime.Unknown, len(args))
+	duplicatePluginConfig := make(map[string]*runtime.Unknown, 0)
 	for i := range args {
-		pluginConfig[args[i].Name] = &args[i].Args
+		name := args[i].Name
+		if _, ok := pluginConfig[name]; ok {
+			duplicatePluginConfig[name] = &args[i].Args
+		}
+		pluginConfig[name] = &args[i].Args
+	}
+	for name, args := range duplicatePluginConfig {
+		klog.Warningf("found duplicate configuration for plugin %q, will use %v", name, args)
```

Comment:
please also highlight which one is enabled :)

Reply:
Thank you @k82cn , but I am not sure how to achieve highlighting? Any examples？